hapi/hedera-protobufs/services/response_code.proto

```diff
@@ -1594,20 +1594,6 @@ enum ResponseCodeEnum {
      */
     INVALID_TOKEN_IN_PENDING_AIRDROP = 369;
 
-    /**
-     * A scheduled transaction configured to wait for expiry to execute was given
-     * an expiry time not strictly after the time at which its creation reached
-     * consensus.
-     */
-    SCHEDULE_EXPIRY_MUST_BE_FUTURE = 370;
-
-    /**
-     * A scheduled transaction configured to wait for expiry to execute was given
-     * an expiry time too far in the future after the time at which its creation
-     * reached consensus.
-     */
-    SCHEDULE_EXPIRY_TOO_LONG = 371;
-
```

Comment:
Is it okay to delete these even if they were cherry-picked and merged?

Reply:
Are we sure that these error codes will never be used?

They have not been made public yet. If we want to delete them, we also have to delete them in 0.57. We should also adjust the values of the following entries.

Reply:
They are duplicated with already existing once - `SCHEDULE_EXPIRATION_TIME_MUST_BE_HIGHER_THAN_CONSENSUS_TIME` and `SCHEDULE_EXPIRATION_TIME_TOO_FAR_IN_FUTURE`.

At first I adjusted the following values but I though it would be better to not change them and reverted back the change.

Reply:
When you make any .proto changes in hedera-services, please make sure you also change in hedera-protobufs.

Reply:
@netopyr the values are adjusted and a PR for the removal of 0.57 is created: https://github.com/hashgraph/hedera-services/pull/16924

@iwsimon these enums were not moved into `hedera-protobufs` at all. No change is needed

Reply:
Okay, I removed the changes from this PR and created a separate one so we can merge it faster and cherry-pick it in 0.57